### PR TITLE
Refresh comment feed

### PR DESCRIPTION
**Pull request by RETprojects**
The comment feed on the workshopping page now shows the new comment immediately after submitting the comment.

### CODE REVIEW

File: apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/CommentForm.js [hunk 1]
```diff
@@ -20,42 +20,51 @@ function CommentForm(props) {
...
     e.preventDefault();
-    var data = { author: userData.username, idea_card: selectedCard, text: textChange.trim() };
+    var data = {
+      text: textChange.trim(),
+      idea_card: selectedCard.id.toString(),
+      author: userData.name,
+      user: userData.id.toString(),
+    };
 
     try {
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/CommentForm.js:39]: can we remove the outer try/catch block here? it looks like it just contains another try/catch block, so it doesn't seem to be serving any purpose here

File: apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/DisplayComments.js [hunk 2]
```diff
@@ -1,28 +1,43 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import Comment from './SingleComment';
-import { cleanDataList } from '../../../../utils/StrapiHelper';
-import axios from 'axios';
+import { cleanData } from '../../../../utils/StrapiHelper';
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/DisplayComments.js:3]: can we remove this import? it doesn't look like it's being used

File: packages/UI/src/context/UserDataContext.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ function useUserDataHook() {
         console.log('Fetching...');
         setUserData({
           id: currentUser.id,
-          name: currentUser.profile.displayName,
+          name: currentUser.profile.name,
```
Comment (dbradham) [inline packages/UI/src/context/UserDataContext.js:42]: is this change needed?

**Review by dbradham — APPROVED** [review]

**Comment by dbradham** [comment]
@RETprojects this is ready to merge but the branch contains conflicts. Please resolve the conflicts or let me know if you are uncertain of how to solve them.

**Comment by RETprojects** [comment]
@dbradham I should keep the changes from my branch in CommentForm, right?
